### PR TITLE
Adding GHO to ethereum tokens

### PR DESCRIPTION
**Pull request by xmc2**
This PR adds the GHO token to the Tokens spell.

### CODE REVIEW

File: models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 1]
```diff
@@ -28502,4 +28502,5 @@ FROM (VALUES
         ,(0xef4928e4a36628408419f0efd99228907b19ace5, 'DogeKing2.0', 9)
         ,(0xeed47eb03a2be37dc103a35bffda92e319e3253a, 'PIPI', 9)
         ,(0x44ae257f8fa2591e6def7d3c7d7bd20f62e3c4dc, '3.0PEPE', 9)
+        ,(0x40d16fc0246ad3160ccc09b8d0d3a2cd28ae6c2f, 'GHO',18)
```
Comment (Hosuke) [inline models/tokens/ethereum/tokens_ethereum_erc20.sql:28505]: https://etherscan.io/token/0x40d16fc0246ad3160ccc09b8d0d3a2cd28ae6c2f

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @xmc2

**Review by jeff-dude — CHANGES_REQUESTED** [review]
thank you @xmc2 -- for the duration of the migration, can you look to add the token to both the dunesql and spark version of the file? you should see the same file with `_legacy.sql` in the same directory.

example here #3747 

note that spark one still contains the quotes on the address

**Comment by xmc2** [comment]
@jeff-dude - updated. Thanks!

**Comment by xmc2** [comment]
Thanks @jeff-dude 
Replace with #3821

**Pull request closed without merging**